### PR TITLE
Feature/basic skeleton

### PR DESCRIPTION
**Pull request by pazaricha**
This version is just an initial layout skeleton with some design and the only thing that's dynamic is importing the restaurants. I intend to add more things and later re-do this with redux. 

A live version of this PR can be seen here: https://paz-weeat-client.herokuapp.com/

**Note: keep in mind the API is on Heroku so it might take a couple of seconds until it shows some restaurants. I plan on adding a spinner there while it's loading later on in the project.**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,28 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Header from './containers/Header/Header';
+import RestaurantsAndMap from './containers/RestaurantsAndMap/RestaurantsAndMap';
+import axios from './axios';
```
Comment (GeReV) [inline src/App.js:4]: I suggest using `https://github.com/github/fetch`. It polyfills a browser capability if it's missing and sort of makes Axios redundant.

**Review by GeReV — APPROVED** [review]
Looks pretty good.

File: src/components/RestaurantIcon/RestaurantIcon.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './RestaurantIcon.css';
+
+const cuisineIconsMap = {
+  'American': 'R',
```
Comment (tomerdesserww) [inline src/components/RestaurantIcon/RestaurantIcon.js:6]: There is a wired coupling between this list and the server list,
I think it's better if it'll sit in the DB too (as part of the metadata maybe ?)

File: src/components/RestaurantsList/RestaurantsList.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.restaurant-list {
```
Comment (tomerdesserww) [inline src/components/RestaurantsList/RestaurantsList.scss:1]: restaurants-list
and also, what will happened when overflow? please set it to scroll

**Review by tomerdesserww — APPROVED** [review]
Very good start

**Pull request merged**